sdk/go/internal/engineconn/unix/unix.go

```diff
@@ -27,15 +26,16 @@ func New(u *url.URL) (engineconn.EngineConn, error) {
 }
 
 func (c *Unix) Connect(ctx context.Context, cfg *engineconn.Config) (*http.Client, error) {
-	if cfg.Workdir != "" {
-		return nil, errors.New("workdir not supported on unix hosts")
-	}
-	if cfg.ConfigPath != "" {
-		return nil, errors.New("config path not supported on unix hosts")
-	}
-	if cfg.NoExtensions {
-		return nil, errors.New("no extensions is not supported on unix hosts")
-	}
+	// FIXME: These are necessary for dagger-in-dagger but do not work.
```

Comment:
@vito @sipsma Because we set a unix:// DAGGER_HOST when running dagger in dagger, I had a test panic'ing (`ExampleHost_Workdir`).

I commented this out -- any alternatives?

Reply:
Hm we could put the examples behind a build flag and skip them in CI. As long as they still run by default when I run `go test` on my machine that should be OK. So CI sets flag, `examples_test.go` skips with flag present.

https://stackoverflow.com/a/24036237

We should be able to remove it with #3671 but there's still work to do there.